app/views/waste_exemptions_engine/pdfs/certificate.html.erb

```diff
@@ -1,192 +1,199 @@
+                  <% presenter.registration_exemptions.each do |registration_exemption| %>
+                    <tr class="govuk-table__row">
+                      <th class="govuk-table__header" scope="row"><%= "#{registration_exemption.exemption.code}: #{registration_exemption.exemption.summary}" %></th>
+                      <td class="govuk-table__cell"><%= "Expires on #{registration_exemption.expires_on.to_formatted_s(:day_month_year)}" %></td>
+                    </tr>
+                  <% end %>
+                </tbody>
+              </table>
+            </div>
+
+            <h2 class="govuk-!-margin-top-9 govuk-heading-m"><%=t ".registration_details.contact_details.heading" %></h2>
+            <div class="table-wrapper">
+              <table class="govuk-table">
+                <tbody class="govuk-table__body">
+                  <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="row"><%=t ".registration_details.contact_details.name" %></th>
+                    <td class="govuk-table__cell"> <%= presenter.applicant_name %> </td>
+                  </tr>
+                  <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="row"><%=t ".registration_details.contact_details.business_telephone" %></th>
+                    <td class="govuk-table__cell"> <%= presenter.applicant_phone %> </td>
+                  </tr>
+                  <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="row"><%=t ".registration_details.contact_details.business_email" %></th>
+                    <td class="govuk-table__cell"> <%= presenter.applicant_email %> </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
 
+            <h2 class="govuk-!-margin-top-9 govuk-heading-m"><%=t ".registration_details.business_details.heading" %></h2>
+            <div class="table-wrapper">
+              <table class="govuk-table">
+                <tbody class="govuk-table__body">
+                  <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="row"><%=t ".registration_details.business_details.business_type" %></th>
+                    <td class="govuk-table__cell"> <%= presenter.business_type.capitalize %> </td>
+                  </tr>
+                  <% if presenter.partnership? %>
+                    <tr class="govuk-table__row">
+                      <th class="govuk-table__header" scope="row">
+                        <%=t ".registration_details.business_details.partners" %>
+                      </th>
+                      <td class="govuk-table__cell"> <%= presenter.partners_names %> </td>
+                    </tr>
+                  <% else %>
+                    <tr class="govuk-table__row">
+                      <th class="govuk-table__header" scope="row"><%=t ".registration_details.business_details.responsible_person" %></th>
```

Comment:
I normally try and make a clear distinction between code review and QA. So I'll focus on quality and consistency in the review, and whether it actually does what it's supposed to in QA.

From a code perspective this is fine but I can (as long as I have understood the code correctly 😁 ) immediately see it will fail QA.

The labels are still referring to **Person responsible ...**. However this will only make sense in the context of a sole trader. If the registrant is Waitrose this will read as

> **Person responsible for these waste operations**: Waitrose Ltd

So as this is still not working for all contexts and clearly so, I don't there is any point approving and merging until we have rectified this issue.

Reply:
nice! Is this knowledge of all cases you have that I am missing! Let's discuss it further when we got a chance :)